package.json

```diff
@@ -184,5 +184,9 @@
       "tokenRef": "GITHUB_AUTH"
     }
   },
-  "trackingCode": "UA-49225444-1"
+  "trackingCode": "UA-49225444-1",
+  "volta": {
```

Comment:
Default yarn these days is yarn 4 -- (not sure if npm or volta did that in the start), 
but using yarn 4 with yarn 1 projects is :grimacing: so for volta users, I set node to min-supported, and set the yarn version to 1